### DISCUSSION TITLE
Do I need VS to create a game?

### DISCUSSION THREAD

**Discussion by ghost** [premake/premake-core#2246] [Q&A]
My question was this, is it possible to create a game using only **premake**, without having to install VS?

**Comment by samsinsane** [comment]
Yes, you can make a game without installing VS. Not sure what you plan on using instead, but I'm sure if you know the tools well enough you'll be fine. You may need to use one of the community modules for Premake, or write your own, to support this alternative setup though.

**Comment by tritao** [comment]
You can use this: https://github.com/mstorsjo/llvm-mingw/
